todo/src/App.js

```diff
@@ -1,28 +1,55 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import TodoForm from './components/TodoForm';
+import TodoList from './components/TodoList';
+import { connect } from 'react-redux';
+import { addTodo, toggleTodo, clearCompleted } from './actions';
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      task: ''
+    }
+  }
```

Comment:
A reminder that since no `props` are used to set anything in your initial state it's not necessary to go through constructor and super, and because you're not doing any function binding or the like within your constructor either, it can actually be eliminated entirely. You can delete the whole constructor here and just write `state={task: '}` and it can be referenced the same way with `this.state` throughout the component